src/extensions/boosts.py

```diff
@@ -0,0 +1,12 @@
+import interactions as discord
+import main
+
+class boosts(discord.Extension):
+    def __init__(self, client):
+        self.client: discord.Client = client
+
+    @discord.listen()
+    async def on_message(self, event: discord.events.MessageCreate):
+        if event.message.type == 8:
```

Comment:
Message type `8` is the normal "`<name> has boosted the server`" message, but there are 3 other boost message types, `9`, `10` & `11` (https://discord.com/developers/docs/resources/channel#message-object-message-types) which you should listen for too